docs/guides/crud-actions.md

```diff
@@ -175,6 +189,16 @@ The following CRUD Action attributes are all optional.
   Defaults to `true`. This property works with actions with any of the CRUD
   action types.
 
+- `mergeListIds` *(Boolean)*: When a list is supplied, this lets you control
+  whether or not the new list of IDs replaces or gets merged into the existing
+  list of IDs for that list. When `true`, it will protect against duplicate
+  IDs being added. Defaults to `true`. This only applies for successful read
```

Comment:
> Defaults to `true`

Might be worthwhile having this up by `(Boolean)` for quick glance

Reply:
If we do this, let's do it all at once in a separate PR, so that all default values are consistent throughout the docs. Open an issue for that if you're interested?